<launch>
  <arg name="launch_prefix" default=""/>
  <arg name="camera_name" default="/back_camera/color"/>
  <arg name="image_topic" default="image_raw"/>
  <arg name="queue_size" default="1"/>

  <node pkg="apriltag_ros" type="apriltag_ros_continuous_node" name="apriltag_ros_continuous_node" clear_params="true" output="screen" launch-prefix="$(arg launch_prefix)">
    <remap from="image_rect" to="$(arg camera_name)/$(arg image_topic)"/>
    <remap from="camera_info" to="$(arg camera_name)/camera_info"/>

    <param name="publish_tag_detections_image" type="bool" value="true"/>
    <param name="queue_size" type="int" value="$(arg queue_size)"/>

    <rosparam file="$(find test_autodock)/params/tag_detection.yaml" command="load"/>
  </node>
</launch>